packages/core/src/types/module-builder.ts

```diff
@@ -174,6 +279,18 @@ export interface IgnitionModuleBuilder {
     options?: StaticCallOptions
   ): StaticCallFuture<ContractNameT, FunctionNameT>;
 
+  /**
+   * Creates a contract instance at the specified address.
```

Comment:
I think this should be more: 
```Creates a future representing an existing deployed contract so that it can be referenced in subsequent futures.```

What do you think?

Reply:
This was 100% a "best effort" attempt on my part as I couldn't really think of a succinct way of saying it.